src/Services/InAppPurchasesService.php

```diff
@@ -33,4 +36,31 @@ public function listPurchases(string $extensionName, Context $context): InAppPur
     {
         return $this->client->listInAppPurchases($extensionName, $context);
     }
+
+    /**
+     * @param Collection<InAppPurchaseCartPositionStruct>|Collection<InAppPurchaseStruct> $allPurchases
```

Comment:
Doesn't it make sense to use your Interface here?
```suggestion
     * @param Collection<InAppPurchaseIdentifiableStruct> $allPurchases
```

Reply:
I had it like that, but then phpstan complained. Changed it again, and saw it's easily solvable with `covariants`